Source/BoundaryConditions/MOSTStress.H

```diff
@@ -316,6 +316,7 @@ struct surface_flux
         u_star_arr(i,j,k) = mdata.kappa * umm_arr(i,j,k) / std::log(mdata.zref / z0_arr(i,j,k));
         do {
             ustar = u_star_arr(i,j,k);
+            // TODO: calculate virtual temperature flux if we have q*
```

Comment:
Could we use the `Tv` mean you now compute in MOSTAverage here? I.e., if a valid moisture model exists, when we call the iterator we just pass the `Tv` plane average instead of the `T` plane average. This would be clean and efficient; just not sure if it's perfectly consistent with the theory?